README.md

```diff
@@ -549,7 +549,13 @@ The test suite depends on a running postgres instance being available on localho
 
 Then, run `rake spec` to run the tests. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
 
-To install this gem onto your local machine, run `bundle exec rake install`.
+To install this gem onto your local machine:
+
+1. Build the gem by running `gem build activestash.gemspec`.
```

Comment:
```suggestion
1. Build the gem by running `rake gem`.
```

Ah, this might be why your gems were ending up in the repo root, rather than `/pkg`.

Reply:
yep you're right, using rake gem, the file goes into /pkg. I'll update the readme and also remove .gem from the gitignore file.